conf/wdio-browserstack-parallel.conf.js

```diff
@@ -0,0 +1,56 @@
+var defaults = require("./wdio.conf.js");
+var _ = require("lodash");
+
+var overrides = {
+  browserstackLocal: false,
```

Comment:
Since browserstack wdio service will not be used this line of code can be removed